experimental/generation/generator/packages/library/src/testGenerator.ts

```diff
@@ -63,6 +63,12 @@ export async function generateTest(path: string, dialog: string, output: string,
             ++userCount
             responses = 0
             description = `Response to input ${userCount}`
+        } else if (isSettingRandomValue(record)) {
+            script.push({
+                $kind: 'Microsoft.Test.CustomEvent', 
+                name: 'SetTestOptions',
+                value: `{randomValue : ${record.value.randomValue}}`
```

Comment:
>`{randomValue : ${record.value.randomValue} [](start = 23, length = 43)

Value should be the whole record.value.  This will pick up randomValue/randomSeed and anything else added.